File: Calculadora/calculadora.swift
print("1° número: ")
var num1 = Double(readLine()!)!
print("Operação: ")
var op = readLine()!
print("2° número: ")
var num2 = Double(readLine()!)!

if op == "+"{
print(num1+num2)
}

if op == "-"{
print(num1-num2)
}

if op == "*"{
print(num1*num2)
}

if op == "/"{
print(num1/num2)
}

else {
print("Tá errado")
}

File: Calculadora/calculadora.cpp
#include <iostream>
#include <string>
using namespace std;

int main() {
  
  int num1, num2;
  char operador;

  cout << "1° valor: ";
  cin >> num1;
  cout << "Operação: ";
  cin >> operador;
  cout << "2° valor: ";
  cin >> num2;

  if (operador == '+'){
    cout << endl;
    cout << num1 + num2 << endl;
  }

  if (operador == '*'){
    cout << endl;
    cout << num1 * num2 << endl;
  }

  if (operador == '-'){
    cout << endl;
    cout << num1 - num2 << endl;
  }

  if (operador == '/'){
    cout << endl;
    cout << num1 / num2 << endl;
  }
}